Arduino/MiP_ProMini_Pack_Library/src/mip.cpp

```diff
@@ -1814,7 +1816,90 @@ int8_t MiP::rawGetGameMode(MiPGameMode& mode)
 
     // Restart the game mode now that we have successfully retrieved it.
     rawSetGameMode(mode);
-    
+
+    return MIP_ERROR_NONE;
+}
+
+
+
+void MiP::setUserData(uint8_t address, uint8_t userData)
```

Comment:
Might be easier for the user if you switched the first parameter from being an address to being an offset and only accept values between 0x00 and 0x0F. You could then add 0x20 before calling down to the raw APIs. You can use MIP_ASSERT() to verify that they keep the offset between 0x00 an 0x0F.

Since you are only calling your verifiedSetUserData() function from one place, you can probably just pull that code up into this function directly and get rid of verifiedSetUserData().